packages/services/hmi-server/src/main/java/software/uncharted/terarium/hmiserver/resources/dataservice/SimulationResource.java

```diff
@@ -61,6 +59,9 @@ public class SimulationResource implements SnakeCaseResource {
 	@Channel("simulationStatus")
 	Emitter<SimulationIntermediateResultsCiemss> simulationStatusEmitter;
 
+	@Inject
+	@Channel("scimlQueue") Publisher<JsonObject> scimlQueueStream;
```

Comment:
Is there a reason we are reading this as a JsonObject rather than a specified object like: `SimulationIntermediateResultsJulia`

Reply:
Yeah, I mentioned it in the PR description!

Reply:
I understand it will be changed in the near future but i think its worth having everything set up and just updating said class when it needs to be updated
What are your thoughts on this?

Reply:
I have tried deserializing the response to a named class as you suggested but I couldn't get it working properly. So that was another reason I went to `JsonObject` for the time being. I don't want that to be a blocker to landing this PR so I'd like to set up `SimulationIntermediateResultsJulia` up in a separate branch if that's cool.
If you have a strong preference though I can try to make the change here!